templates/_error.tmpl

```diff
@@ -0,0 +1,25 @@
+{% extends 'base_index.html' %}
+
+{% block title %}{{error_code}}: {{error_message}}{% endblock %}
+
+{% block meta_description %}{{error_message}}{% endblock %}
+
+{% set is_paper = True %}
+
+{% block content %}
+  <div class="p-strip">
+    <div class="row p-section">
```

Comment:
@lyubomir-popov as per @mtruj013 's comment : 
> Should we change this to a deep strip as well? Again, from Vanilla: "Sections should not be nested inside each other or inside the strips."

Reply:
yes well spotted. Please correct it.